src/core/ext/transport/chttp2/server/chttp2_server.c

```diff
@@ -49,10 +49,10 @@ typedef struct {
   grpc_closure tcp_server_shutdown_complete;
   grpc_closure *server_destroy_listener_done;
   grpc_handshake_manager *pending_handshake_mgrs;
-} server_state;
+} server_state_t;
```

Comment:
I believe that `_t` suffixes are reserved for system types.  And in any case, we don't use this suffix for any other struct type in our code, so let's be consistent.  I think the original name is fine here.

Reply:
Same thing here. Would need to change either the type name or the variable name. Adding _t to the suffix seemed like a simple fix here, since this is a typedef in a source file, and should not pollute the global namespace.

Reply:
In C, everything's in the global namespace.  Even if this isn't visible outside of this file, it could still conflict with something added in a future version of the system C headers.

Instead of changing the type name, let's change the field name on line 55 from `server_state` to `svr_state`.

Reply:
done!